tutorials/4 Analysis/C. Parametric sweeps/4.23 Impedance and scattering Z S Y matrices.ipynb

```diff
@@ -228,96 +228,169 @@
   },
   {
    "cell_type": "markdown",
-   "id": "b0e47077",
+   "id": "4bfca232",
    "metadata": {},
    "source": [
-    "## 2. Render the qubit from Metal into the HangingResonators design in Ansys. <br>Must open Ansys HFSS at this point. \n",
+    "## 2. Render the qubit from Metal into the HangingResonators design in Ansys. <br>ScatteringImpedanceSim will  open the simulation software. Then will connect, activate the design, add a setup.  \n",
     "\n",
-    "**Wait for Ansys to fully open before proceeding.**<br> If necessary, also close any Ansys popup windows.\n",
+    "Review and update the setup. For driven modal you will need to define not only the simulation convergence parameters, but also the frequency sweep.\n",
     "\n",
-    "### Registration occurs automatically within the method \"sweep_one_option_get_drivenmodal_solution_data\". \n",
+    "Customizable parameters and default values for HFSS (driven  modal):\n",
     "\n",
-    "### Must have an open Ansys project and HFSS eigenmode design before executing render_design. Can do it manually or automate it. \n",
-    "\n",
-    "Manually: To create a new Ansys project, click \"New\", the leftmost icon right below the file menu. \n",
-    "\n",
-    "Automate: Execute the command <hfss.open_ansys()> in cell below only if Ansys is not already open.  Use the cells below to open the Ansys application.  \n",
-    "\n",
-    "### The method \"sweep_one_option_get_drivenmodal_solution_data\" will connect to ansys, activate a HFSS design, add a setup, and insert frequency-sweep under the setup."
+    "    freq_ghz=5 (simulation frequency)\n",
```

Comment:
This will quickly get out of date. What do you think about just relying on the call to the sim setup for the list of defaults and maybe a url to the docs for the details about what each parameter is?

(this comments also applies to the following markdown cell (setup_sweep)

Reply:
I hear you, there are now cells that show the options that can be selected.  But there is not text to explain it.  This notebook example is for ScatteringImpedanceSim using hfss.  Those parameters are not expected to change anytime soon.  Also, there is text explaining the parameters in parentheses.  

Going forward, for additional renderer examples, we can skip adding documentation to the notebook, and rely on the documentation in the renderer.